common/lib/xmodule/xmodule/course_module.py

```diff
@@ -1316,11 +1317,21 @@ def grading_context(self):
         except UndefinedContext:
             module = self
 
+        graded_location_block_types = set(
+            cat for (cat, xblock_class) in XBlock.load_classes() if (
+                getattr(xblock_class, 'has_score', True) or getattr(xblock_class, 'has_children', True)
+            )
+        )
```

Comment:
I moved this to be a lazy attribute on the `CourseDescriptor`. Trying to set it at the module level caused import errors.